frontend/views/Login.tsx

```diff
@@ -49,7 +49,7 @@ function Login(props: Props): React.ReactElement {
   const [password, setPassword] = useState<string>("");
   const [passwordConfirmation, setPasswordConfirmation] = useState<string>("");
   const route = useRoute();
-  const token = route.params ? route.params.token : null;
+  const token = props.route.params ? props.route.params.token || "" : "";
```

Comment:
`props.route.params` might be undefined that's why this was throwing an error

Reply:
There has to be a better syntax for this, I'm not really sure why the previous code wouldn't work. If params was undefined then params.token would be undefined and if it was it'd use empty string as a default. 

I guess just make it what it previously was

```ts
const token = props.route.params ? props.route.params.token : "";
```